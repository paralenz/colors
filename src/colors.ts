export const colors = {
  crabRed: '#681603',
  oceanGreen: '#203033',
  oceanBlue: '#152228',
  coral: '#ff604c',
  turquoise: '#a5d7b7',
  lagoon: '#6787b7',
  deepRed: '#440706',
  deepGreen: '#131d1e',
  deepSea: '#000000',
  pirateYellow: '#e9ff6b',
  shrimp: '#ffc6c2',
  lightTurquoise: '#e6f3e6',
  aqua: '#c3d7ee',
  white: '#ffffff',
  black: '#000000',
  rentalStateAvailable: '#abff57',
  rentalStateHandedOut: '#e2e2e2',
  rentalStateRented: '#579aff',
  rentalStateHandedIn: '#ff575c',
  rentalStateUnlocked: '#9DBEF1',
  rentalStateUploaded: '#a6dc91',
  sessionDot: '#ffc116',
  destroy: '#e74c3c',
  seafoam: '#ccebe0',
  transparent: 'transparent'
}
