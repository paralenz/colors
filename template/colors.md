
<!-- DO NOT EDIT THIS FILE! This is autogenerated -->
| Name | Color |
| crabRed | ![](https://via.placeholder.com/15/#681603/000000?text=+) #681603  |
| oceanGreen | ![](https://via.placeholder.com/15/#203033/000000?text=+) #203033  |
| oceanBlue | ![](https://via.placeholder.com/15/#152228/000000?text=+) #152228  |
| coral | ![](https://via.placeholder.com/15/#ff604c/000000?text=+) #ff604c  |
| turquoise | ![](https://via.placeholder.com/15/#a5d7b7/000000?text=+) #a5d7b7  |
| lagoon | ![](https://via.placeholder.com/15/#6787b7/000000?text=+) #6787b7  |
| deepRed | ![](https://via.placeholder.com/15/#440706/000000?text=+) #440706  |
| deepGreen | ![](https://via.placeholder.com/15/#131d1e/000000?text=+) #131d1e  |
| deepSea | ![](https://via.placeholder.com/15/#000000/000000?text=+) #000000  |
| pirateYellow | ![](https://via.placeholder.com/15/#e9ff6b/000000?text=+) #e9ff6b  |
| shrimp | ![](https://via.placeholder.com/15/#ffc6c2/000000?text=+) #ffc6c2  |
| lightTurquoise | ![](https://via.placeholder.com/15/#e6f3e6/000000?text=+) #e6f3e6  |
| aqua | ![](https://via.placeholder.com/15/#c3d7ee/000000?text=+) #c3d7ee  |
| white | ![](https://via.placeholder.com/15/#ffffff/000000?text=+) #ffffff  |
| black | ![](https://via.placeholder.com/15/#000000/000000?text=+) #000000  |
| rentalStateAvailable | ![](https://via.placeholder.com/15/#abff57/000000?text=+) #abff57  |
| rentalStateHandedOut | ![](https://via.placeholder.com/15/#e2e2e2/000000?text=+) #e2e2e2  |
| rentalStateRented | ![](https://via.placeholder.com/15/#579aff/000000?text=+) #579aff  |
| rentalStateHandedIn | ![](https://via.placeholder.com/15/#ff575c/000000?text=+) #ff575c  |
| rentalStateUnlocked | ![](https://via.placeholder.com/15/#9DBEF1/000000?text=+) #9DBEF1  |
| rentalStateUploaded | ![](https://via.placeholder.com/15/#a6dc91/000000?text=+) #a6dc91  |
| sessionDot | ![](https://via.placeholder.com/15/#ffc116/000000?text=+) #ffc116  |
| destroy | ![](https://via.placeholder.com/15/#e74c3c/000000?text=+) #e74c3c  |
| transparent | ![](https://via.placeholder.com/15/transparent/000000?text=+) transparent  |