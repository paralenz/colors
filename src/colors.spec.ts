import { colors } from './colors'

describe('colors', () => {
  it('should match snapshot', () => {
    expect(colors).toMatchSnapshot()
  });
});