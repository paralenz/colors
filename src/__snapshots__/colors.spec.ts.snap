// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`colors should match snapshot 1`] = `
Object {
  "aqua": "#c3d7ee",
  "black": "#000000",
  "coral": "#ff604c",
  "crabRed": "#681603",
  "deepGreen": "#131d1e",
  "deepRed": "#440706",
  "deepSea": "#000000",
  "destroy": "#e74c3c",
  "lagoon": "#6787b7",
  "lightTurquoise": "#e6f3e6",
  "oceanBlue": "#152228",
  "oceanGreen": "#203033",
  "pirateYellow": "#e9ff6b",
  "rentalStateAvailable": "#abff57",
  "rentalStateHandedIn": "#ff575c",
  "rentalStateHandedOut": "#e2e2e2",
  "rentalStateRented": "#579aff",
  "rentalStateUnlocked": "#9DBEF1",
  "rentalStateUploaded": "#a6dc91",
  "seafoam": "#ccebe0",
  "sessionDot": "#ffc116",
  "shrimp": "#ffc6c2",
  "transparent": "transparent",
  "turquoise": "#a5d7b7",
  "white": "#ffffff",
}
`;
